english/sleep/make-another-wish/siri/marshmallow-walls.md

```diff
@@ -0,0 +1,7 @@
+Sure! I have a few choices available for your search.
+
+You choose [YouTube](https://www.youtube.com/watch?v=NSkycYIUnLg)  
+This song reminded me of something... perhaps, I should [sing a song](../../../sing/sing.md) and try to remember how it all started...
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.